### PR TITLE
data: gowthae

### PR DESCRIPTION
**Pull request by gowthae**
creating new profile : gowthae

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6973"></a>

### CODE REVIEW

File: data/gowthae.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+{
+  "name": "Gowtham E",
+  "displayStatsPublic":true,
+  "type": "personal",
+  "bio": "I'm a Software Developer",
+  "avatar": "https://github.com/gowthae.png",
```
Comment (Panquesito7) [inline data/gowthae.json:7]: I believe this field is no longer required. The avatar is automatically fetched from your profile.
```suggestion
```

File: data/gowthae.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+{
+  "name": "Gowtham E",
+  "displayStatsPublic":true,
```
Comment (Panquesito7) [inline data/gowthae.json:4]: Same as this one (stats are only available to you, AFAIK).
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by harshsingh32 — COMMENTED** [review]
Thanks for contribution.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**